bin/pt-kill

```diff
@@ -8011,6 +8030,11 @@ pt-kill does not provide any safeguards so code carefully!
 
 It is permissible for the code to have side effects (to alter C<$event>).
 
+=item --db-type
+
+type: string
+
+mysql; tidb
```

Comment:
Here should be the default value for mysql. Please check other options where the default value is defined.